<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customers</class>
 <widget class="QMainWindow" name="Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clients</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableCustomers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>530</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="French" country="Switzerland"/>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adresse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NPA</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Localité</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Téléphone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mobile</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
